R/LTFH.R

```diff
@@ -46,12 +46,17 @@ LTFH <- function(covmat, phenos, N=10000, K = 0.05, s_val = 0, min_run=400, all_
 
   current_liabil = rep(s_val, k)
 
+  # det er muligvis en præference ting, men jeg er ikke så stor fan af at hardcode jeres burn-in på den her måde.
+  # jeg synes egentligt i skulle sætte den som argument i LTFH.
+  # i kunne faktisk også overveje bare at ændre navnet på denne funktion, da den, i al sin simpelhed, bare sampler
+  # fra en trunkeret flerdimensionel normal fordeling. Når i så estimere den genetiske liability osv,
+  # kan i kalde den funktion LTFH
   start_run = 500
   s = 0
   p = 0
 
   while(TRUE) {
-    p = p +1
+    p = p +1 # jeg er ikke lige helt med på formålet med p.
```

Comment:
bare for god ordensskyld, så kan i også kommentere på enkelte linjer eller ændringer. Så kan man drøfte ting ned i mindste detalje.